### DISCUSSION TITLE
How does the "Waiting for Cirrus CI" step work?

### DISCUSSION THREAD

**Discussion by jbarlow83** [bigcat88/pillow_heif#105] [Q&A] [answered]
I'm trying to set up GitHub + Cirrus CI wheel building similar to what this repo has.

I don't understand how `lewagon/wait-on-check-action` knows to ask Cirrus CI when it's done building; or if it goes the other way, at what point Cirrus CI signals to the GitHub Checks API that it's done. How does this work? Are there any GitHub level settings not reflected in config files?

**Comment by bigcat88** [comment, marked as answer]
Hello. It checks every `wait-interval` for the name `check-name`
In cirrus_wheels.yml you can see that name of cirrus task.
After step is finished `Download artifacts` try downloading artifacts(by alias).

There is no special settings for this.
